ide/gsf.testrunner.ui/src/org/netbeans/modules/gsf/testrunner/ui/api/TestsuiteNode.java

```diff
@@ -69,9 +70,7 @@ public class TestsuiteNode extends AbstractNode {
      * See Issue #175430
      * See Issue #188632
      */
-    static final int MAX_MSG_LINE_LENGTH = 
-          Integer.getInteger("testrunner.max.msg.line.length", //NOI18N
-                             isGTK() || isJDK8() ? 120 : Integer.MAX_VALUE);
+    static final int MAX_MSG_LINE_LENGTH = Integer.getInteger("testrunner.max.msg.line.length", 250);
```

Comment:
original logic was a bit questionable. The GTK LAF check did make somewhat sense but the freezing x-server issue is reproducible even on FlatLAF, the JDK 8 check is weird too. I was always scared to move the mouse over the test window while running NB tests :)